bundles/tampere/content-editor/view/SideContentEditor.js

```diff
@@ -931,8 +937,8 @@ Oskari.clazz.define('Oskari.tampere.bundle.content-editor.view.SideContentEditor
         },
         _storeFormData: function () {
             var me = this;
-            $(".getinforesult_table tr input").each(function (index) {
-                me.currentData.features[0][index+1] = $(this).val();
+            jQuery(".getinforesult_table tr input").each(function (index) {
```

Comment:
Limit the search to inputs in this functionality with: me.mainPanel.find('.getinforesult_table tr input')